projects/packages/connection/src/class-rest-connector.php

```diff
@@ -92,6 +92,35 @@ public function __construct( Manager $connection ) {
 			)
 		);
 
+		// Disconnect site.
+		register_rest_route(
+			'jetpack/v4',
+			'/connection',
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => __CLASS__ . '::disconnect_site',
+				'permission_callback' => __CLASS__ . '::disconnect_site_permission_check',
+				'args'                => array(
+					'isActive' => array(
+						'description'       => __( 'Set to false will trigger the site to disconnect.', 'jetpack' ),
+						'validate_callback' => function ( $value ) {
+							$is_valid = false === $value;
+							if ( ! $is_valid ) {
```

Comment:
would this be easier to read as a single line in the if statement

if( false !== $value ) 

nit-pick but as is_valid isn't used elsewhere having it pulled out makes me question what is valid.